cmd/skyeye/main.go

```diff
@@ -105,6 +171,9 @@ var skyeye = &cobra.Command{
 	Long:    "Skyeye uses real-time telemetry data from TacView to provide Ground-Controlled Intercept service over SimpleRadio-Standalone.",
 	Example: strings.Join(
 		[]string{
+			"Custom Config Path",
```

Comment:
The `"  " +` on line 177 is a hack to make the examples indent properly in the help text, so move it up to line 174

Reply:
done